docs/tutorials/helloworldintro.md

```diff
@@ -44,40 +53,55 @@ See the [Install MLAPI](../migration/installation.md) guide to install the MLAPI
 1. Inside the `NetworkManager` component tab, locate the  `NetworkTransport` field. 
 1. Click "Select Transport".
 1. Select `UnetTransport`.
+1. Save your scene.
+
+
+<iframe src="https://www.youtube.com/embed/ZYEUWzsXEoY?playlist=ZYEUWzsXEoY&loop=1&&autoplay=0&controls=1&showinfo=0&mute=1"   width="854px"
+        height="480px" className="video-container" frameborder="0" position="relative" allow="accelerometer; autoplay; loop; playlist; clipboard-write; encrypted-media; gyroscope; picture-in-picture"  allowfullscreen=""></iframe>
+
+
 
-  ![selecttransaportani](/img/selecting-transport.gif)
+## Creating an object to spawn for each connected player
 
 1. Create 3D Object->Capsule (name it **Player**). 
 1. Add a `NetworkObject` component.
 1. Click the Assets folder.
 1. Create a new Folder and call it **Prefabs**.
 1. Make **Player** a prefab by dragging it to **Prefabs** folder you just created.
-
-  ![createplayer prefab](/img/createprefab.gif)
-
 1. Delete **Player** from scene.
 
   :::tip
   We remove **Player**, because we will be using the network library to spawn the player. The library cannot track objects that start in the scene.
   :::
 
-1. Add Player prefab to `NetworkPrefabs` list inside of `NetworkManager`.
-1. Select **Default Player Prefab**.
-
-  ![addingdefaultprefab](/img/default-player-prefab.gif)
+1. Select `NetworkManager`.
```

Comment:
is this a button/ui element? would be bolded not coded.

Reply:
Done